federator/integrationTest/integrationTest.js

```diff
@@ -12,6 +12,10 @@ const abiSideToken = require('../../bridge/abi/SideToken.json');
 const abiAllowTokens = require('../../bridge/abi/AllowTokens.json');
 const abiMultiSig = require('../../bridge/abi/MultiSigWallet.json');
 
+const abiNFTBridge = require('../../bridge/abi/NFTBridge.json');
```

Comment:
We should create a new file for the NFT integration test, something like /integrationTest/nftIntegrationTest.js